DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Geometry/EnvelopBoxToSpeckleConverter.cs

```diff
@@ -0,0 +1,56 @@
+using ArcGIS.Core.Geometry;
+using Speckle.Converters.Common.Objects;
+using Speckle.Converters.Common;
+using ArcGIS.Desktop.Mapping;
+using Objects.Primitive;
+
+namespace Speckle.Converters.ArcGIS3.Geometry;
+
+public class EnvelopToSpeckleConverter : IRawConversion<Envelope, SOG.Box>
+{
+  private readonly IConversionContextStack<Map, Unit> _contextStack;
+  private readonly IRawConversion<MapPoint, SOG.Point> _pointConverter;
+
+  public EnvelopToSpeckleConverter(
+    IConversionContextStack<Map, Unit> contextStack,
+    IRawConversion<MapPoint, SOG.Point> pointConverter
+  )
+  {
+    _contextStack = contextStack;
+    _pointConverter = pointConverter;
+  }
+
+  public SOG.Box RawConvert(Envelope target)
+  {
+    MapPoint pointMin = new MapPointBuilderEx(
+      target.XMin,
+      target.YMin,
+      target.ZMin,
+      _contextStack.Current.Document.SpatialReference
+    ).ToGeometry();
+    MapPoint pointMax = new MapPointBuilderEx(
+      target.XMax,
+      target.YMax,
+      target.ZMax,
+      _contextStack.Current.Document.SpatialReference
+    ).ToGeometry();
```

Comment:
this can be refactored into a private method and used twice

Reply:
It's min vs max, I could create 2 methods to get those coordinates and unpack them Python style (*list) inside MapPointBuilderEx method, but didn't find a good way to do it on C#